pkg/lightning/backend/local/region_job.go

```diff
@@ -436,6 +436,10 @@ func (local *Backend) doWrite(ctx context.Context, j *regionJob) error {
 		allPeers = append(allPeers, peer)
 	}
 	dataCommitTS := j.ingestData.GetTS()
+	if dataCommitTS == 0 {
+		intest.Assert(false, "data commitTS is 0")
```

Comment:
```suggestion
```

Reply:
I added it because DDL can retry the whole ingest task. Without this line, the newly added unit test can pass, because when the second task has zero TS it will fail, and DDL continues from checkpoint.

Reply:
> DDL can retry the whole ingest task.

shouldn't this be a fatal error?